Library/Formula/signing-party.rb

```diff
@@ -201,3 +202,15 @@ def install
  # Organiser contact
  printf "%80s\n\n\n" "$org"
  # Title
+
+--- a/caff/caff	2016-02-01 21:59:58.000000000 +0100
++++ b/caff/caff	2016-02-01 22:00:42.000000000 +0100
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!